Avoid inline literals in titles (#384)
doctrine/data-fixtures: docs/en/how-to/loading-fixtures.rst

------------------------

Fixture classes have two requirements:

* They must implement ``Doctrine\Common\DataFixtures\FixtureInterface``.

> * If they have a constructor, that constructor should not accept any
>   arguments.

Reviewer: ```suggestion
* If they have a constructor, that constructor should be invokable
  without arguments.
```
